spec/spec_helper.rb

```diff
@@ -30,7 +32,7 @@
   config.before :context, type: :feature do
     Capybara.app = SequenceServer.init
     Capybara.server = :webrick
-    Capybara.default_max_wait_time = 10
+    Capybara.default_max_wait_time = 25
```

Comment:
This is causing test to take ages on my Mac. This is probably the reason Travis build failed for your PR: ["The job exceeded the maximum time limit for jobs, and has been terminated."](https://travis-ci.org/wurmlab/sequenceserver/jobs/597657844)

Reply:
I will changed that, so on master is always 10 seconds. However, it could be worth mentioning in documentation about this possibility ?